### PR TITLE
Added ability to setup text labels content

### PR DESCRIPTION
**Pull request by bsekachev**
### Motivation and context
Sber request

image

### How has this been tested?
Manual testing

### Checklist

- [x] I submit my changes into the `develop` branch
- [x] I have added description of my changes into [CHANGELOG](https://github.com/opencv/cvat/blob/develop/CHANGELOG.md) file
- [ ] I have updated the [documentation](
  https://github.com/opencv/cvat/blob/develop/README.md#documentation) accordingly
- [ ] I have added tests to cover my changes
- [ ] I have linked related issues ([read github docs](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))
- [x] I have increased versions of npm packages if it is necessary ([cvat-canvas](https://github.com/opencv/cvat/tree/develop/cvat-canvas#versioning),
[cvat-core](https://github.com/opencv/cvat/tree/develop/cvat-core#versioning), [cvat-data](https://github.com/opencv/cvat/tree/develop/cvat-data#versioning) and [cvat-ui](https://github.com/opencv/cvat/tree/develop/cvat-ui#versioning))

### License

- [x] I submit _my code changes_ under the same [MIT License](
  https://github.com/opencv/cvat/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.
- [x] I have updated the license header for each file (see an example below)

```python
# Copyright (C) 2021 Intel Corporation
#
# SPDX-License-Identifier: MIT
```

### CODE REVIEW

**Comment by bsekachev** [comment]
Hi guys, 

@TOsmanov Will you be able to add a piece of documentation?
@dvkruchinin Could you please add a test  (or update case 111)  to cover the feature?
 @nstolyarov It was implemented to satisfy Sber request as we discussed on email. Can you take a look into how it works?

**Comment by dvkruchinin** [comment]
> Could you please add a test (or update case 111) to cover the feature?

Hi. Sure. The test will be updated.

**Comment by bsekachev** [comment]
@ActiveChooN Could you please review?

**Comment by TOsmanov** [comment]
> @TOsmanov Will you be able to add a piece of documentation?

@bsekachev Hi, sure.

File: cvat-canvas/src/typescript/canvasModel.ts [hunk 1]
```diff
@@ -649,14 +654,21 @@ export class CanvasModelImpl extends MasterImpl implements CanvasModel {
             this.data.configuration.displayAllText = configuration.displayAllText;
         }
 
-        if (typeof configuration.textFontSize === 'number') {
+        if (typeof configuration.textFontSize === 'number' && configuration.textFontSize >= 8) {
```
Comment (ActiveChooN) [inline cvat-canvas/src/typescript/canvasModel.ts:657]: Maybe const `MIN_TEXT_FONT_SIZE` worths adding. Can be helpful in the future.

Comment (bsekachev) [reply cvat-canvas/src/typescript/canvasModel.ts:657]: Yep, maybe

Comment (bsekachev) [reply cvat-canvas/src/typescript/canvasModel.ts]: Added

**Review by ActiveChooN — APPROVED** [review]

**Pull request merged**